pages/api/home/home.tsx

```diff
@@ -55,7 +52,7 @@ const Home = ({
   const { t } = useTranslation('chat');
   const { getModels } = useApiService();
   const { getModelsError } = useErrorService();
-  const [initialRender, setInitialRender] = useState<boolean>(true);
+  const isInitialRender = useRef<boolean>(true);
```

Comment:
useState was dead code. Needed a useRef because (I think) Next.js was causing 2 conversations to be started on page load due to SSR (i.e. server made one, then client made one)